### PR TITLE
updates for 3.0.0 changelog

### PR DESCRIPTION
**Pull request by danrosenthal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 ---
 
+## 3.0.0 - 2018-11-09
+
+### New components
+
+We've released a suite of new components that, when combined, form the application frame of a standalone (or non-embedded) Polaris app.
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
We’ve released a suite of new components that, when combined, form the application frame of a standalone (or non-embedded) Polaris app.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 ---
 
+## 3.0.0 - 2018-11-09
+
+### New components
+
+We’ve released a suite of new components that, when combined, form the application frame of a standalone (or non-embedded) Polaris app.
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
We’ve released a suite of new components that, when combined, form the application frame of a stand-alone (or non-embedded) Polaris app.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Changed `Form` to default the `method` to `post` in order to prevent accidental leaking of form details
+- Added support for boolean type on Choice error prop
+- Changed the esnext folder to contain individual, minimally transpiled JavaScript component files, as well as raw style and image assets
+- Added `onPortalCreated` prop to `Portal`
+- Improved consistency of `Badge` styling
+- Explicitly specifying `list-style` on `List`
+
+### Bug fixes
+
+- Fixed console error and used new ref syntax in `DataTable` (thanks to ([@duythien0912](https://github.com/duythien0912)) for the ([original issue](https://github.com/Shopify/polaris-react/issues/403)))
```
Comment (BPScott) [inline CHANGELOG.md]:
```suggestion
- Fixed console error and used new ref syntax in `DataTable` (thanks to [@duythien0912](https://github.com/duythien0912) for the [original issue](https://github.com/Shopify/polaris-react/issues/403))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Fixed the ability to upload multiple files even when `allowedMultiple` prop is false
+- Fixed `Datatable` so it resizes with new content (thanks [@andrewpye](https://github.com/andrewpye) for the [original issue](https://github.com/Shopify/polaris-react/issues/387))
+- Fixed `RangeSlider` linear-gradient so it doesn't break the css build (thanks [@Ankitjasoliya](https://github.com/Ankitjasoliya) and [@nerfologist](https://github.com/nerfologist) for the [original issue](https://github.com/Shopify/polaris-react/issues/441))
+- Fixed issue in `Page`, where styling wasn't being applied correctly to Page Actions
+- Removed unnecessary bindings on `Modal`s `onClose` prop. [original issue](https://github.com/Shopify/polaris-react/issues/441))
+- Rearranged `primaryFooterAction` and `secondaryFooterAction` in `Card` (thanks [@sivakumar-kailasam](https://github.com/sivakumar-kailasam) for the [original issue](https://github.com/Shopify/polaris-react/issues/551))
+
+### Breaking changes
```
Comment (BPScott) [inline CHANGELOG.md]: Should breaking changes go before bug-fixes and enhancements? It feels more important

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+
+- Added padding top and bottom on `Card.Section` when set to full width
+- Fixed `Portal` rendering by using `componentDidMount` lifecycle hook as opposed to `componentWillMount`;
+- Fixed an issue where clicking a `Link` without a `url` in a form would implicitly submit the form. `Link` can no longer submit forms. Use `<Button submit>` instead.
+- Renamed the `Keys` enum to align with Shopify naming standards. It is now singular and the properties are in PascalCase. Replace `import {Keys} from '@shopify/polaris'` with `import {Key} from '@shopify/polaris'` and change the casing of the properties, e.g. replace `Keys.DOWN_ARROW` with `Key.DownArrow`
+- Added !important to `display: none` in `@print-hidden` mixin
+
+#### Embedded apps
+
+- Upgraded to the Shopify App Bridge and removed the EASDK
```
Comment (BPScott) [inline CHANGELOG.md]:
```suggestion
- Use the Shopify App Bridge in favor of the EASDK. The EASDK has been removed
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Added new `resourceType`, `initialQuery`, and `showVariants` props to `ResourcePicker` which aligns with the Shopify App Bridge API
+- Moved embedded `ResourcePicker`, `Modal`, and `Page` to the main bundle and removed the embedded bundle. Imports from `'@shopify/polaris/embedded'` will no longer work, use `'@shopify/polaris'` instead
+- Made the `shopOrigin` prop on `AppProvider` optional. It’s now provided by default. If you do provide a `shopOrigin` it now needs to be given without the `'https://'` per the Shopify App Bridge API.
+- Updated `onSelection` prop on `ResourcePicker`. The shape of the `selectPayload` data has changed and the product `id` is now a `gid`. For example, `/9019381572` is now `gid://shopify/Product/9019381572`. We offer [@shopify/admin-graphql-api-utilities](https://www.npmjs.com/package/@shopify/admin-graphql-api-utilities) to help compose and parse `gid` from Shopify admin
+- Updated default values for `ResourcePicker` props to align with the Shopify App Bridge. Set prop `showHidden`, `allowMultiple={false}`, and `showVariants={false}` to get the previous default behavior
```
Comment (BPScott) [inline CHANGELOG.md]:
```suggestion
- Updated default values for `ResourcePicker` props to align with the Shopify App Bridge. Set the props `showHidden`, `allowMultiple={false}`, and `showVariants={false}` to get the previous default behavior
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Removed the `Alert` component, use `Modal` with `message` prop instead
+- Replaced `easdk` on React context with `appBridge`. Access it via `this.context.polaris.appBridge`.
+- Removed `this.context.easdk.startLoading()` and `this.context.easdk.stopLoading()`, use the `Loading` component instead
+- Removed `this.context.easdk.showFlashNotice()`, use the `Toast` component instead
+- Removed `this.context.easdk.pushState()`, use the [Shopify App Bridge `History` action](https://help.shopify.com/en/api/embedded-apps/app-bridge/actions/navigation/history) instead. The `History` action requires passing the `appBridge` instance which is accessible via `this.context.polaris.appBridge`
+- Removed `this.context.easdk.redirect()`, use the [Shopify App Bridge `Redirect` action](https://help.shopify.com/en/api/embedded-apps/app-bridge/actions/navigation/redirect) instead. The `Redirect` action requires passing the `appBridge` instance which is accessible via `this.context.polaris.appBridge`
+
+#### License
```
Comment (BPScott) [inline CHANGELOG.md]: We don't want people to miss this. I reckon pull it up higher. I can't decide on directly above or below breaking changes (technically the licence change is a breaking change).

File: CHANGELOG.md [hunk 8]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Changed `Form` to default the `method` to `post` in order to prevent accidental leaking of form details
+- Added support for boolean type on Choice error prop
+- Changed the esnext folder to contain individual, minimally transpiled JavaScript component files, as well as raw style and image assets
+- Added `onPortalCreated` prop to `Portal`
+- Improved consistency of `Badge` styling
+- Explicitly specifying `list-style` on `List`
+
+### Bug fixes
+
+- Fixed console error and used new ref syntax in `DataTable` (thanks to [@duythien0912](https://github.com/duythien0912)) for the [original issue](https://github.com/Shopify/polaris-react/issues/403))
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Fixed console error and used new ref syntax in `DataTable` (thanks to [@duythien0912](https://github.com/duythien0912) for the [original issue](https://github.com/Shopify/polaris-react/issues/403))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Explicitly specifying `list-style` on `List`
+
+### Bug fixes
+
+- Fixed console error and used new ref syntax in `DataTable` (thanks to [@duythien0912](https://github.com/duythien0912) for the [original issue](https://github.com/Shopify/polaris-react/issues/403))
+- Fixed the ability to upload multiple files even when `allowedMultiple` prop is false
+- Fixed `Datatable` so it resizes with new content (thanks [@andrewpye](https://github.com/andrewpye) for the [original issue](https://github.com/Shopify/polaris-react/issues/387))
+- Fixed `RangeSlider` linear-gradient so it doesn't break the css build (thanks [@Ankitjasoliya](https://github.com/Ankitjasoliya) and [@nerfologist](https://github.com/nerfologist) for the [original issue](https://github.com/Shopify/polaris-react/issues/441))
+- Fixed issue in `Page`, where styling wasn't being applied correctly to Page Actions
+- Removed unnecessary bindings on `Modal’s `onClose` prop
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Removed unnecessary bindings on the `Modal`’s `onClose` prop
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Fixed `Datatable` so it resizes with new content (thanks [@andrewpye](https://github.com/andrewpye) for the [original issue](https://github.com/Shopify/polaris-react/issues/387))
+- Fixed `RangeSlider` linear-gradient so it doesn't break the css build (thanks [@Ankitjasoliya](https://github.com/Ankitjasoliya) and [@nerfologist](https://github.com/nerfologist) for the [original issue](https://github.com/Shopify/polaris-react/issues/441))
+- Fixed issue in `Page`, where styling wasn't being applied correctly to Page Actions
+- Removed unnecessary bindings on the `Modal`’s `onClose` prop
+- Rearranged `primaryFooterAction` and `secondaryFooterAction` in `Card` (thanks [@sivakumar-kailasam](https://github.com/sivakumar-kailasam) for the [original issue](https://github.com/Shopify/polaris-react/issues/551))
+
+### Breaking changes
+
+- Added padding top and bottom on `Card.Section` when set to full width
+- Fixed `Portal` rendering by using `componentDidMount` lifecycle hook as opposed to `componentWillMount`;
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Fixed `Portal` rendering by using `componentDidMount` lifecycle hook as opposed to `componentWillMount`
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Fixed an issue where clicking a `Link` without a `url` in a form would implicitly submit the form. `Link` can no longer submit forms. Use `<Button submit>` instead.
+- Renamed the `Keys` enum to align with Shopify naming standards. It is now singular and the properties are in PascalCase. Replace `import {Keys} from '@shopify/polaris'` with `import {Key} from '@shopify/polaris'` and change the casing of the properties, e.g. replace `Keys.DOWN_ARROW` with `Key.DownArrow`
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Renamed the `Keys` enum to align with Shopify naming standards. It is now singular and the properties are in PascalCase. Replace `import {Keys} from '@shopify/polaris'` with `import {Key} from '@shopify/polaris'` and change the casing of the properties, e.g. replace `Keys.DOWN_ARROW` with `Key.DownArrow`.
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Added !important to `display: none` in `@print-hidden` mixin
+
+#### Embedded apps
+
+- Upgraded to the Shopify App Bridge and removed the EASDK
+- Added Shopify App Bridge support to new components `Toast` and `Loading`
+- Added `target` prop to all actions which get passed to the Shopify App Bridge
+- Added new `size` and `message` props to `Modal` which aligns with the Shopify App Bridge API
+- Added new `resourceType`, `initialQuery`, and `showVariants` props to `ResourcePicker` which aligns with the Shopify App Bridge API
+- Moved embedded `ResourcePicker`, `Modal`, and `Page` to the main bundle and removed the embedded bundle. Imports from `'@shopify/polaris/embedded'` will no longer work, use `'@shopify/polaris'` instead
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Moved embedded `ResourcePicker`, `Modal`, and `Page` to the main bundle and removed the embedded bundle. Imports from `'@shopify/polaris/embedded'` will no longer work, use `'@shopify/polaris'` instead.
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Made the `shopOrigin` prop on `AppProvider` optional. It’s now provided by default. If you do provide a `shopOrigin` it now needs to be given without the `'https://'` per the Shopify App Bridge API.
+- Updated `onSelection` prop on `ResourcePicker`. The shape of the `selectPayload` data has changed and the product `id` is now a `gid`. For example, `/9019381572` is now `gid://shopify/Product/9019381572`. We offer [@shopify/admin-graphql-api-utilities](https://www.npmjs.com/package/@shopify/admin-graphql-api-utilities) to help compose and parse `gid` from Shopify admin
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Updated `onSelection` prop on `ResourcePicker`. The shape of the `selectPayload` data has changed and the product `id` is now a `gid`. For example, `/9019381572` is now `gid://shopify/Product/9019381572`. We offer [@shopify/admin-graphql-api-utilities](https://www.npmjs.com/package/@shopify/admin-graphql-api-utilities) to help compose and parse `gid` from Shopify admin.
```

File: CHANGELOG.md [hunk 14]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Updated default values for `ResourcePicker` props to align with the Shopify App Bridge. Set the props `showHidden`, `allowMultiple={false}`, and `showVariants={false}` to get the previous default behavior
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Updated default values for `ResourcePicker` props to align with the Shopify App Bridge. Set the props `showHidden`, `allowMultiple={false}`, and `showVariants={false}` to get the previous default behavior.
```

File: CHANGELOG.md [hunk 15]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Updated `target` prop type related to embedded apps, use `'APP'`, `'ADMIN_PATH'`, or `'REMOTE'`
+- Removed `icon` prop from `Page`. Upload your app’s icon in the Shopify Partners dashboard “App setup” section instead
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Removed `icon` prop from `Page`. Upload your app’s icon in the Shopify Partners dashboard “App setup” section instead.
```

File: CHANGELOG.md [hunk 16]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Removed the `Alert` component, use `Modal` with `message` prop instead
+- Replaced `easdk` on React context with `appBridge`. Access it via `this.context.polaris.appBridge`.
+- Removed `this.context.easdk.startLoading()` and `this.context.easdk.stopLoading()`, use the `Loading` component instead
+- Removed `this.context.easdk.showFlashNotice()`, use the `Toast` component instead
+- Removed `this.context.easdk.pushState()`, use the [Shopify App Bridge `History` action](https://help.shopify.com/en/api/embedded-apps/app-bridge/actions/navigation/history) instead. The `History` action requires passing the `appBridge` instance which is accessible via `this.context.polaris.appBridge`
+- Removed `this.context.easdk.redirect()`, use the [Shopify App Bridge `Redirect` action](https://help.shopify.com/en/api/embedded-apps/app-bridge/actions/navigation/redirect) instead. The `Redirect` action requires passing the `appBridge` instance which is accessible via `this.context.polaris.appBridge`
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Removed `this.context.easdk.redirect()`, use the [Shopify App Bridge `Redirect` action](https://help.shopify.com/en/api/embedded-apps/app-bridge/actions/navigation/redirect) instead. The `Redirect` action requires passing the `appBridge` instance which is accessible via `this.context.polaris.appBridge`.
```

File: CHANGELOG.md [hunk 17]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Removed `products` prop from `ResourcePicker`, use `resourceType="Product"` instead
+- Removed `collections` prop from `ResourcePicker`, use `resourceType="Collection"` instead
+- Removed `width` and `height` props from `Modal`, use `size` instead
+- Removed `debug` prop from `AppProvider`, use [Redux DevTools](https://github.com/reduxjs/redux-devtools) instead. Redux DevTools also has [browser extensions](https://github.com/zalmoxisus/redux-devtools-extension).
+- Removed the `Alert` component, use `Modal` with `message` prop instead
+- Replaced `easdk` on React context with `appBridge`. Access it via `this.context.polaris.appBridge`.
+- Removed `this.context.easdk.startLoading()` and `this.context.easdk.stopLoading()`, use the `Loading` component instead
+- Removed `this.context.easdk.showFlashNotice()`, use the `Toast` component instead
+- Removed `this.context.easdk.pushState()`, use the [Shopify App Bridge `History` action](https://help.shopify.com/en/api/embedded-apps/app-bridge/actions/navigation/history) instead. The `History` action requires passing the `appBridge` instance which is accessible via `this.context.polaris.appBridge`
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Removed `this.context.easdk.pushState()`, use the [Shopify App Bridge `History` action](https://help.shopify.com/en/api/embedded-apps/app-bridge/actions/navigation/history) instead. The `History` action requires passing the `appBridge` instance which is accessible via `this.context.polaris.appBridge`.
```

File: CHANGELOG.md [hunk 18]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Made the `shopOrigin` prop on `AppProvider` optional. It’s now provided by default. If you do provide a `shopOrigin` it now needs to be given without the `'https://'` per the Shopify App Bridge API.
+- Updated `onSelection` prop on `ResourcePicker`. The shape of the `selectPayload` data has changed and the product `id` is now a `gid`. For example, `/9019381572` is now `gid://shopify/Product/9019381572`. We offer [@shopify/admin-graphql-api-utilities](https://www.npmjs.com/package/@shopify/admin-graphql-api-utilities) to help compose and parse `gid` from Shopify admin
+- Updated default values for `ResourcePicker` props to align with the Shopify App Bridge. Set the props `showHidden`, `allowMultiple={false}`, and `showVariants={false}` to get the previous default behavior
+- Updated `target` prop type related to embedded apps, use `'APP'`, `'ADMIN_PATH'`, or `'REMOTE'`
+- Removed `icon` prop from `Page`. Upload your app’s icon in the Shopify Partners dashboard “App setup” section instead
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Removed `icon` prop from `Page`. Upload your app’s icon in the Shopify Partners dashboard “App setup” section instead.
```

File: CHANGELOG.md [hunk 19]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Added new `resourceType`, `initialQuery`, and `showVariants` props to `ResourcePicker` which aligns with the Shopify App Bridge API
+- Moved embedded `ResourcePicker`, `Modal`, and `Page` to the main bundle and removed the embedded bundle. Imports from `'@shopify/polaris/embedded'` will no longer work, use `'@shopify/polaris'` instead
+- Made the `shopOrigin` prop on `AppProvider` optional. It’s now provided by default. If you do provide a `shopOrigin` it now needs to be given without the `'https://'` per the Shopify App Bridge API.
+- Updated `onSelection` prop on `ResourcePicker`. The shape of the `selectPayload` data has changed and the product `id` is now a `gid`. For example, `/9019381572` is now `gid://shopify/Product/9019381572`. We offer [@shopify/admin-graphql-api-utilities](https://www.npmjs.com/package/@shopify/admin-graphql-api-utilities) to help compose and parse `gid` from Shopify admin
+- Updated default values for `ResourcePicker` props to align with the Shopify App Bridge. Set the props `showHidden`, `allowMultiple={false}`, and `showVariants={false}` to get the previous default behavior
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Updated default values for `ResourcePicker` props to align with the Shopify App Bridge. Set the props `showHidden`, `allowMultiple={false}`, and `showVariants={false}` to get the previous default behavior.
```

File: CHANGELOG.md [hunk 20]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Added Shopify App Bridge support to new components `Toast` and `Loading`
+- Added `target` prop to all actions which get passed to the Shopify App Bridge
+- Added new `size` and `message` props to `Modal` which aligns with the Shopify App Bridge API
+- Added new `resourceType`, `initialQuery`, and `showVariants` props to `ResourcePicker` which aligns with the Shopify App Bridge API
+- Moved embedded `ResourcePicker`, `Modal`, and `Page` to the main bundle and removed the embedded bundle. Imports from `'@shopify/polaris/embedded'` will no longer work, use `'@shopify/polaris'` instead
+- Made the `shopOrigin` prop on `AppProvider` optional. It’s now provided by default. If you do provide a `shopOrigin` it now needs to be given without the `'https://'` per the Shopify App Bridge API.
+- Updated `onSelection` prop on `ResourcePicker`. The shape of the `selectPayload` data has changed and the product `id` is now a `gid`. For example, `/9019381572` is now `gid://shopify/Product/9019381572`. We offer [@shopify/admin-graphql-api-utilities](https://www.npmjs.com/package/@shopify/admin-graphql-api-utilities) to help compose and parse `gid` from Shopify admin
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Updated `onSelection` prop on `ResourcePicker`. The shape of the `selectPayload` data has changed and the product `id` is now a `gid`. For example, `/9019381572` is now `gid://shopify/Product/9019381572`. We offer [@shopify/admin-graphql-api-utilities](https://www.npmjs.com/package/@shopify/admin-graphql-api-utilities) to help compose and parse `gid` from Shopify admin.
```

File: CHANGELOG.md [hunk 21]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Added !important to `display: none` in `@print-hidden` mixin
+
+#### Embedded apps
+
+- Upgraded to the Shopify App Bridge and removed the EASDK
+- Added Shopify App Bridge support to new components `Toast` and `Loading`
+- Added `target` prop to all actions which get passed to the Shopify App Bridge
+- Added new `size` and `message` props to `Modal` which aligns with the Shopify App Bridge API
+- Added new `resourceType`, `initialQuery`, and `showVariants` props to `ResourcePicker` which aligns with the Shopify App Bridge API
+- Moved embedded `ResourcePicker`, `Modal`, and `Page` to the main bundle and removed the embedded bundle. Imports from `'@shopify/polaris/embedded'` will no longer work, use `'@shopify/polaris'` instead
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Moved embedded `ResourcePicker`, `Modal`, and `Page` to the main bundle and removed the embedded bundle. Imports from `'@shopify/polaris/embedded'` will no longer work, use `'@shopify/polaris'` instead.
```

File: CHANGELOG.md [hunk 22]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 ---
 
+## 3.0.0 - 2018-11-09
+
+### Breaking changes
+
+- Added padding top and bottom on `Card.Section` when set to full width
+- Fixed `Portal` rendering by using `componentDidMount` lifecycle hook as opposed to `componentWillMount`;
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Fixed `Portal` rendering by using `componentDidMount` lifecycle hook as opposed to `componentWillMount`
```

File: CHANGELOG.md [hunk 23]
```diff
@@ -8,6 +8,132 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Fixed an issue where clicking a `Link` without a `url` in a form would implicitly submit the form. `Link` can no longer submit forms. Use `<Button submit>` instead.
+- Renamed the `Keys` enum to align with Shopify naming standards. It is now singular and the properties are in PascalCase. Replace `import {Keys} from '@shopify/polaris'` with `import {Key} from '@shopify/polaris'` and change the casing of the properties, e.g. replace `Keys.DOWN_ARROW` with `Key.DownArrow`
```
Comment (tmlayton) [inline CHANGELOG.md]:
```suggestion
- Renamed the `Keys` enum to align with Shopify naming standards. It is now singular and the properties are in PascalCase. Replace `import {Keys} from '@shopify/polaris'` with `import {Key} from '@shopify/polaris'` and change the casing of the properties, e.g. replace `Keys.DOWN_ARROW` with `Key.DownArrow`.
```

**Pull request merged**